### PR TITLE
Add canned searches

### PR DESCRIPTION
**Pull request by jnowakow**
### Fixed Issues

$ https://github.com/Expensify/App/issues/41625
PROPOSAL: N/A

### Tests

1. Go to Search Page, `Profile > Troubleshoot > New search page`
2. There should be 4 options to pick
3. Only `Expenses` option works because other are not implemented on backend.

- [x] Verify that no errors appear in the JS console

### Offline tests

N/A

### QA Steps

Same as in Tests

- [x] Verify that no errors appear in the JS console

### PR Author Checklist

- [x] I linked the correct issue in the `### Fixed Issues` section above
- [x] I wrote clear testing steps that cover the changes made in this PR
    - [x] I added steps for local testing in the `Tests` section
    - [x] I added steps for the expected offline behavior in the `Offline steps` section
    - [x] I added steps for Staging and/or Production testing in the `QA steps` section
    - [x] I added steps to cover failure scenarios (i.e. verify an input displays the correct error message if the entered data is not correct)
    - [x] I turned off my network connection and tested it while offline to ensure it matches the expected behavior (i.e. verify the default avatar icon is displayed if app is offline)
    - [x] I tested this PR with a [High Traffic account](https://github.com/Expensify/App/blob/main/contributingGuides/CONTRIBUTING.md#high-traffic-accounts) against the staging or production API to ensure there are no regressions (e.g. long loading states that impact usability).
- [x] I included screenshots or videos for tests on [all platforms](https://github.com/Expensify/App/blob/main/contributingGuides/CONTRIBUTING.md#make-sure-you-can-test-on-all-platforms)
- [x] I ran the tests on **all platforms** & verified they passed on:
    - [x] Android: Native
    - [x] Android: mWeb Chrome
    - [x] iOS: Native
    - [x] iOS: mWeb Safari
    - [x] MacOS: Chrome / Safari
    - [x] MacOS: Desktop
- [x] I verified there are no console errors (if there's a console error not related to the PR, report it or open an issue for it to be fixed)
- [x] I followed proper code patterns (see [Reviewing the code](https://github.com/Expensify/App/blob/main/contributingGuides/PR_REVIEW_GUIDELINES.md#reviewing-the-code))
    - [x] I verified that any callback methods that were added or modified are named for what the method does and never what callback they handle (i.e. `toggleReport` and not `onIconClick`)
    - [x] I verified that the left part of a conditional rendering a React component is a boolean and NOT a string, e.g. `myBool && <MyComponent />`.
    - [x] I verified that comments were added to code that is not self explanatory
    - [x] I verified that any new or modified comments were clear, correct English, and explained "why" the code was doing something instead of only explaining "what" the code was doing.
    - [x] I verified any copy / text shown in the product is localized by adding it to `src/languages/*` files and using the [translation method](https://github.com/Expensify/App/blob/4bd99402cebdf4d7394e0d1f260879ea238197eb/src/components/withLocalize.js#L60)
      - [x] If any non-english text was added/modified, I verified the translation was requested/reviewed in #expensify-open-source and it was approved by an internal Expensify engineer. Link to Slack message:
    - [x] I verified all numbers, amounts, dates and phone numbers shown in the product are using the [localization methods](https://github.com/Expensify/App/blob/4bd99402cebdf4d7394e0d1f260879ea238197eb/src/components/withLocalize.js#L60-L68)
    - [x] I verified any copy / text that was added to the app is grammatically correct in English. It adheres to proper capitalization guidelines (note: only the first word of header/labels should be capitalized), and is either coming verbatim from figma or has been approved by marketing (in order to get marketing approval, ask the Bug Zero team member to add the Waiting for copy label to the issue)
    - [x] I verified proper file naming conventions were followed for any new files or renamed files. All non-platform specific files are named after what they export and are not named "index.js". All platform-specific files are named for the platform the code supports as outlined in the README.
    - [x] I verified the JSDocs style guidelines (in [`STYLE.md`](https://github.com/Expensify/App/blob/main/contributingGuides/STYLE.md#jsdocs)) were followed
- [x] If a new code pattern is added I verified it was agreed to be used by multiple Expensify engineers
- [x] I followed the guidelines as stated in the [Review Guidelines](https://github.com/Expensify/App/blob/main/contributingGuides/PR_REVIEW_GUIDELINES.md)
- [x] I tested other components that can be impacted by my changes (i.e. if the PR modifies a shared library or component like `Avatar`, I verified the components using `Avatar` are working as expected)
- [x] I verified all code is DRY (the PR doesn't include any logic written more than once, with the exception of tests)
- [x] I verified any variables that can be defined as constants (ie. in CONST.js or at the top of the file that uses the constant) are defined as such
- [x] I verified that if a function's arguments changed that all usages have also been updated correctly
- [x] If any new file was added I verified that:
    - [x] The file has a description of what it does and/or why is needed at the top of the file if the code is not self explanatory
- [x] If a new CSS style is added I verified that:
    - [x] A similar style doesn't already exist
    - [x] The style can't be created with an existing [StyleUtils](https://github.com/Expensify/App/blob/main/src/styles/utils/index.ts) function (i.e. `StyleUtils.getBackgroundAndBorderStyle(theme.componentBG)`)
- [x] If the PR modifies code that runs when editing or sending messages, I tested and verified there is no unexpected behavior for all supported markdown - URLs, single line code, code blocks, quotes, headings, bold, strikethrough, and italic.
- [x] If the PR modifies a generic component, I tested and verified that those changes do not break usages of that component in the rest of the App (i.e. if a shared library or component like `Avatar` is modified, I verified that `Avatar` is working as expected in all cases)
- [x] If the PR modifies a component related to any of the existing Storybook stories, I tested and verified all stories for that component are still working as expected.
- [x] If the PR modifies a component or page that can be accessed by a direct deeplink, I verified that the code functions as expected when the deeplink is used - from a logged in and logged out account.
- [x] If the PR modifies the UI (e.g. new buttons, new UI components, changing the padding/spacing/sizing, moving components, etc) or modifies the form input styles:
    - [x] I verified that all the inputs inside a form are aligned with each other.
    - [x] I added `Design` label and/or tagged `@Expensify/design` so the design team can review the changes.
- [x] If a new page is added, I verified it's using the `ScrollView` component to make it scrollable when more elements are added to the page.
- [x] If the `main` branch was merged into this PR after a review, I tested again and verified the outcome was still expected according to the `Test` steps.

### Screenshots/Videos
<details>
<summary>Android: Native</summary>

https://github.com/Expensify/App/assets/56261019/3ed5fbc4-2c23-43ac-bc7b-ba1d77c31939

</details>

<details>
<summary>Android: mWeb Chrome</summary>

https://github.com/Expensify/App/assets/56261019/9bc65465-e25b-4343-9759-affa8ca0852f

</details>

<details>
<summary>iOS: Native</summary>

https://github.com/Expensify/App/assets/56261019/ca8511ab-7d42-46f6-8c0b-961d62c193f5

</details>

<details>
<summary>iOS: mWeb Safari</summary>

https://github.com/Expensify/App/assets/56261019/7fa1bf78-eff7-40c3-9ee6-d2c503f93463

</details>

<details>
<summary>MacOS: Chrome / Safari</summary>

https://github.com/Expensify/App/assets/56261019/68c1e1a8-f1df-4757-9a22-b133ba8c1cc4

</details>

<details>
<summary>MacOS: Desktop</summary>

https://github.com/Expensify/App/assets/56261019/b40e94cc-2ddf-4319-99c3-978bfd1a18d9

</details>

### CODE REVIEW

File: assets/images/simple-illustrations/simple-illustration__drafts.svg
Comment (jnowakow) [inline assets/images/simple-illustrations/simple-illustration__drafts.svg:1]: @luacmartins how am I supposed to properly export the illustration from Figma?

File: assets/images/simple-illustrations/simple-illustration__finished.svg
Comment (jnowakow) [inline assets/images/simple-illustrations/simple-illustration__finished.svg:1]: @luacmartins how am I supposed to properly export the illustration from Figma?

File: assets/images/simple-illustrations/simple-illustration__shared.svg
Comment (jnowakow) [inline assets/images/simple-illustrations/simple-illustration__shared.svg:1]: @luacmartins how am I supposed to properly export the illustration from Figma?

**Review by Kicu — APPROVED** [review]
Hey this is pretty straightforward and everything looks good to me.

Can you confirm in `devtools > Network` that after clicking on a different tab the `/search` api call is sent with a different and correct query param? (just drop a screenshot)

**Comment by jnowakow** [comment]
> Can you confirm in `devtools > Network` that after clicking on a different tab the `/search` api call is sent with a different and correct query param? (just drop a screenshot)

@Kicu yup, I believe it works as expected:

image
image
image

**Comment by shawnborton** [comment]
The screenshots look cool, but just wondering how we want to handle this UI in light of Search v2? Thoughts @Expensify/design @JmillsExpensify @luacmartins? Should we be making these as tabs across the top of the content pane view?

**Comment by dannymcclain** [comment]
Oh boy, yeah I think we should 😬

**Comment by dubielzyk-expensify** [comment]
Yeah, that probably makes sense.

**Comment by luacmartins** [comment]
Discussed this with @trjExpensify and @JmillsExpensify but I think we still want to push forward as is, since we're still months out on Search v2 and adding these items to the LHN would allow us to test things in the meantime without the LHN looking really empty and weird. Additionally, we still need to design the tab bar and how its filtering capabilities would work, so this is a good stepping stone in the meantime.

**Comment by JmillsExpensify** [comment]
Yes agreed. I can see both perspectives, but it we move the type filter to the LHN, then it'll basically remain `Expenses` for months, as we aren't bringing other types in the LHN until later phases of the Search v2 implementation, which hasn't yet begun.

**Comment by shawnborton** [comment]
Okay cool, that works for me then.

**Comment by shawnborton** [comment]
Looks good to me, confirming that changing the canned search doesn't actually load results except for the first item of Expenses, right?

File: assets/images/simple-illustrations/simple-illustration__checkmark-circle.svg
Comment (shawnborton) [inline assets/images/simple-illustrations/simple-illustration__checkmark-circle.svg:1]: Did the @Expensify/design export this for you, or did you grab this yourself from Figma? We've had some trouble in the past when contributors grab illustrations themselves so just want to double check.

**Comment by Kicu** [comment]
> ... confirming that changing the canned search doesn't actually load [...]

Correct, backend can right now return only `Expenses`. There is no handling of the different search types yet.

**Comment by dannymcclain** [comment]
The main thing I noticed is that the `check-circle` icon in the Finished nav item is really out of place. And it looks like all the icons in these nav items are `24 x 24px` when they should be `20 x 20px`.

### Check-circle icon 🤨
image

Did that icon get exported properly?

### Nav item icon size 🕵️
CleanShot 2024-05-23 at 09 04 44@2x

Screenshots from the ad hoc build on the left, Figma measurements on the right

**Comment by shawnborton** [comment]
Nice catch! Yeah, I think we (design team) need to supply all of these icons/assets so they scale correctly. Can you export them quickly Danny? Otherwise I am happy to do it too!

**Comment by dannymcclain** [comment]
Looks like several of these are already in the repo:

- Receipt* - assets/images/receipt.svg
- Plane* - assets/images/send.svg
- Pencil* - assets/images/pencil.svg
- **Check circle** - This is a new one: [Check Circle.svg.zip](https://github.com/Expensify/App/files/15418877/Check.Circle.svg.zip)

*These icons look right, just too big as I mentioned in my previous comment

File: assets/images/simple-illustrations/simple-illustration__checkmark-circle.svg
Comment (jnowakow) [reply assets/images/simple-illustrations/simple-illustration__checkmark-circle.svg:1]: I grabbed it by myself. So maybe they should be exported once again by design team.

**Comment by jnowakow** [comment]
Thanks for catching this issue @dannymcclain! I fixed it

**Comment by shawnborton** [comment]
Thanks for updating! Looks like we're ready for review then.

**Comment by luacmartins** [comment]
Assigning @parasharrajat as C+ since you're assigned to the issue

**Comment by luacmartins** [comment]
FYI @jnowakow @parasharrajat we'll hold merging this PR until the backend is implemented.

File: assets/images/simple-illustrations/simple-illustration__checkmark-circle.svg
Comment (luacmartins) [reply assets/images/simple-illustrations/simple-illustration__checkmark-circle.svg:1]: Let's get @Expensify/design to export these assets and use those instead to make sure they follow the expected format

File: package-lock.json [hunk 1]
```diff
@@ -1852,19 +1852,6 @@
...
-      "dependencies": {
-        "@babel/helper-plugin-utils": "^7.18.6"
-      },
-      "engines": {
-        "node": ">=6.9.0"
-      },
-      "peerDependencies": {
-        "@babel/core": "^7.0.0-0"
-      }
-    },
```
Comment (luacmartins) [inline package-lock.json:1867]: Why do we have these changes? I don't see anything changing in `package.json`

File: src/languages/es.ts [hunk 2]
```diff
@@ -324,6 +324,9 @@ export default {
         action: 'Acción',
         expenses: 'Gastos',
         tax: 'Impuesto',
+        shared: 'Compartido',
+        drafts: 'Borradores',
+        finished: 'Finalizado',
```
Comment (luacmartins) [inline src/languages/es.ts:329]: Did these translations get approved internally?

File: src/pages/Search/SearchFilters.tsx [hunk 3]
```diff
@@ -58,8 +80,8 @@ function SearchFilters({query}: SearchFiltersProps) {
                         interactive
                         title={item.title}
                         icon={item.icon}
-                        iconWidth={variables.iconSizeLarge}
-                        iconHeight={variables.iconSizeLarge}
+                        iconWidth={variables.iconSizeNormal}
+                        iconHeight={variables.iconSizeNormal}
```
Comment (luacmartins) [inline src/pages/Search/SearchFilters.tsx:84]: Why this change?

File: src/pages/Search/SearchPage.tsx [hunk 4]
```diff
@@ -27,6 +27,18 @@ function SearchPage({route}: SearchPageProps) {
...
+            title: translate('common.shared'),
+        },
+        drafts: {
+            icon: Illustrations.Pencil,
+            title: translate('common.drafts'),
+        },
+        finished: {
+            icon: Illustrations.CheckmarkCircle,
+            title: translate('common.finished'),
+        },
```
Comment (luacmartins) [inline src/pages/Search/SearchPage.tsx:41]: Let's follow the same code style here, i.e. either keep them to a single line or split into multiple lines

**Review by luacmartins — CHANGES_REQUESTED** [review]
Looking good. Left a few small comments though

**Comment by luacmartins** [comment]
I think we can test the menus now, but the actual pages won't return any data. That's independent from this PR though.

File: src/pages/Search/SearchFilters.tsx (hunk 3 above)
Comment (jnowakow) [reply src/pages/Search/SearchFilters.tsx:84]: Because of [this comment](https://github.com/Expensify/App/pull/42184#issuecomment-2127216129)

File: src/pages/Search/SearchPage.tsx (hunk 4 above)
Comment (jnowakow) [reply src/pages/Search/SearchPage.tsx:41]: It's automatically formatted by prettier

File: package-lock.json (hunk 1 above)
Comment (Kicu) [reply package-lock.json:1867]: I think a PR that just adds some feature and does not modify any packages should not introduce any changes to `package-lock` - even if these changes are on main. That's messy.

I suggest you drop/revert any changes to lockfile @jnowakow

File: assets/images/simple-illustrations/simple-illustration__checkmark-circle.svg
Comment (jnowakow) [reply assets/images/simple-illustrations/simple-illustration__checkmark-circle.svg:1]: @dannymcclain you exported the check-circle for me. Are you able to provide me also those two illustrations?

File: assets/images/simple-illustrations/simple-illustration__send-money.svg
Comment (shawnborton) [inline assets/images/simple-illustrations/simple-illustration__send-money.svg:1]: This one already exists in the repo, right? I think we can use the existing asset and delete this.

File: assets/images/simple-illustrations/simple-illustration__checkmark-circle.svg
Comment (shawnborton) [inline assets/images/simple-illustrations/simple-illustration__checkmark-circle.svg:1]: here you go: [simple-illustration__checkmarkcircle.svg.zip](https://github.com/Expensify/App/files/15466410/simple-illustration__checkmarkcircle.svg.zip)

Comment (shawnborton) [reply assets/images/simple-illustrations/simple-illustration__checkmark-circle.svg:1]: I think Danny already uploaded this one for ya, but here it is again: [check-circle.svg.zip](https://github.com/Expensify/App/files/15466416/check-circle.svg.zip)

File: src/pages/Search/SearchPage.tsx (hunk 4 above)
Comment (luacmartins) [reply src/pages/Search/SearchPage.tsx:42]: Oh weird that it formats `all` differently since they all seem to have the same line length 🤔

**Review by luacmartins — APPROVED** [review]
LGTM

**Comment by luacmartins** [comment]
@parasharrajat this is for the new search page. You can access it via `Profile > Troubleshoot > New search page`

**Review by parasharrajat — APPROVED** [review]
## Reviewer Checklist

- [x] I have verified the author checklist is complete (all boxes are checked off).
- [x] I verified the correct issue is linked in the `### Fixed Issues` section above
- [x] I verified testing steps are clear and they cover the changes made in this PR
    - [x] I verified the steps for local testing are in the `Tests` section
    - [x] I verified the steps for Staging and/or Production testing are in the `QA steps` section
    - [x] I verified the steps cover any possible failure scenarios (i.e. verify an input displays the correct error message if the entered data is not correct)
    - [x] I turned off my network connection and tested it while offline to ensure it matches the expected behavior (i.e. verify the default avatar icon is displayed if app is offline)
- [x] I checked that screenshots or videos are included for tests on [all platforms](https://github.com/Expensify/App/blob/main/contributingGuides/CONTRIBUTING.md#make-sure-you-can-test-on-all-platforms)
- [x] I included screenshots or videos for tests on [all platforms](https://github.com/Expensify/App/blob/main/contributingGuides/CONTRIBUTING.md#make-sure-you-can-test-on-all-platforms)
- [x] I verified tests pass on **all platforms** & I tested again on:
    - [x] Android: Native
    - [x] Android: mWeb Chrome
    - [x] iOS: Native
    - [x] iOS: mWeb Safari
    - [x] MacOS: Chrome / Safari
    - [x] MacOS: Desktop
- [x] If there are any errors in the console that are unrelated to this PR, I either fixed them (preferred) or linked to where I reported them in Slack
- [x] I verified proper code patterns were followed (see [Reviewing the code](https://github.com/Expensify/App/blob/main/contributingGuides/PR_REVIEW_GUIDELINES.md#reviewing-the-code))
    - [x] I verified that any callback methods that were added or modified are named for what the method does and never what callback they handle (i.e. `toggleReport` and not `onIconClick`).
    - [x] I verified that the left part of a conditional rendering a React component is a boolean and NOT a string, e.g. `myBool && <MyComponent />`.
    - [x] I verified that comments were added to code that is not self explanatory
    - [x] I verified that any new or modified comments were clear, correct English, and explained "why" the code was doing something instead of only explaining "what" the code was doing.
    - [x] I verified any copy / text shown in the product is localized by adding it to `src/languages/*` files and using the [translation method](https://github.com/Expensify/App/blob/4bd99402cebdf4d7394e0d1f260879ea238197eb/src/components/withLocalize.js#L60)
    - [x] I verified all numbers, amounts, dates and phone numbers shown in the product are using the [localization methods](https://github.com/Expensify/App/blob/4bd99402cebdf4d7394e0d1f260879ea238197eb/src/components/withLocalize.js#L60-L68)
    - [x] I verified any copy / text that was added to the app is grammatically correct in English. It adheres to proper capitalization guidelines (note: only the first word of header/labels should be capitalized), and is approved by marketing by adding the `Waiting for Copy` label for a copy review on the original GH to get the correct copy.
    - [x] I verified proper file naming conventions were followed for any new files or renamed files. All non-platform specific files are named after what they export and are not named "index.js". All platform-specific files are named for the platform the code supports as outlined in the README.
    - [x] I verified the JSDocs style guidelines (in [`STYLE.md`](https://github.com/Expensify/App/blob/main/contributingGuides/STYLE.md#jsdocs)) were followed
- [x] If a new code pattern is added I verified it was agreed to be used by multiple Expensify engineers
- [x] I verified that this PR follows the guidelines as stated in the [Review Guidelines](https://github.com/Expensify/App/blob/main/contributingGuides/PR_REVIEW_GUIDELINES.md)
- [x] I verified other components that can be impacted by these changes have been tested, and I retested again (i.e. if the PR modifies a shared library or component like `Avatar`, I verified the components using `Avatar` have been tested & I retested again)
- [x] I verified all code is DRY (the PR doesn't include any logic written more than once, with the exception of tests)
- [x] I verified any variables that can be defined as constants (ie. in CONST.js or at the top of the file that uses the constant) are defined as such
- [x] If a new component is created I verified that:
    - [x] A similar component doesn't exist in the codebase
    - [x] All props are defined accurately and each prop has a `/** comment above it */`
    - [x] The file is named correctly
    - [x] The component has a clear name that is non-ambiguous and the purpose of the component can be inferred from the name alone
    - [x] The only data being stored in the state is data necessary for rendering and nothing else
    - [x] For Class Components, any internal methods passed to components event handlers are bound to `this` properly so there are no scoping issues (i.e. for `onClick={this.submit}` the method `this.submit` should be bound to `this` in the constructor)
    - [x] Any internal methods bound to `this` are necessary to be bound (i.e. avoid `this.submit = this.submit.bind(this);` if `this.submit` is never passed to a component event handler like `onClick`)
    - [x] All JSX used for rendering exists in the render method
    - [x] The component has the minimum amount of code necessary for its purpose, and it is broken down into smaller components in order to separate concerns and functions
- [x] If any new file was added I verified that:
    - [x] The file has a description of what it does and/or why is needed at the top of the file if the code is not self explanatory
- [x] If a new CSS style is added I verified that:
    - [x] A similar style doesn't already exist
    - [x] The style can't be created with an existing [StyleUtils](https://github.com/Expensify/App/blob/main/src/utils/index.ts) function (i.e. `StyleUtils.getBackgroundAndBorderStyle(theme.componentBG`)
- [x] If the PR modifies code that runs when editing or sending messages, I tested and verified there is no unexpected behavior for all supported markdown - URLs, single line code, code blocks, quotes, headings, bold, strikethrough, and italic.
- [x] If the PR modifies a generic component, I tested and verified that those changes do not break usages of that component in the rest of the App (i.e. if a shared library or component like `Avatar` is modified, I verified that `Avatar` is working as expected in all cases)
- [x] If the PR modifies a component related to any of the existing Storybook stories, I tested and verified all stories for that component are still working as expected.
- [x] If the PR modifies a component or page that can be accessed by a direct deeplink, I verified that the code functions as expected when the deeplink is used - from a logged in and logged out account.
- [x] If the PR modifies the form input styles:
    - [x] I verified that all the inputs inside a form are aligned with each other.
    - [x] I added `Design` label so the design team can review the changes.
- [x] If a new page is added, I verified it's using the `ScrollView` component to make it scrollable when more elements are added to the page.
- [x] If the `main` branch was merged into this PR after a review, I tested again and verified the outcome was still expected according to the `Test` steps.
- [x] I have checked off every checkbox in the PR reviewer checklist, including those that don't apply to this PR.

:ribbon: :eyes: :ribbon:  C+ reviewed

File: src/languages/es.ts [hunk 5]
```diff
@@ -324,6 +324,9 @@ export default {
         action: 'Acción',
         expenses: 'Gastos',
         tax: 'Impuesto',
+        shared: 'Compartido',
```
Comment (luacmartins) [inline src/languages/es.ts:327]:
```suggestion
        shared: 'Compartidos',
```

File: src/languages/es.ts (hunk 2 above)
Comment (luacmartins) [inline src/languages/es.ts:329]:
```suggestion
        finished: 'Finalizados',
```

**Review by luacmartins — CHANGES_REQUESTED** [review]

File: src/languages/es.ts (hunk 2 above)
Comment (jnowakow) [reply src/languages/es.ts:329]: @luacmartins [they got approved](https://swmansion.slack.com/archives/C01GTK53T8Q/p1716884578368599)

File: src/pages/Search/SearchPage.tsx (hunk 4 above)
Comment (jnowakow) [reply src/pages/Search/SearchPage.tsx]: Okey, it occurs that it was my bad and prettier allows two options 🤷

File: package-lock.json (hunk 1 above)
Comment (jnowakow) [reply package-lock.json]: I reverted that change as @Kicu advised

File: assets/images/simple-illustrations/simple-illustration__send-money.svg
Comment (jnowakow) [reply assets/images/simple-illustrations/simple-illustration__send-money.svg:1]: It exists in `docs` folder so at least I have to copy it to `assets`. When I do so the illustration doesn't look good: 
image

Compared to how it looks with my export: 
image

File: assets/images/simple-illustrations/simple-illustration__checkmark-circle.svg
Comment (jnowakow) [reply assets/images/simple-illustrations/simple-illustration__checkmark-circle.svg:1]: Thanks!

File: assets/images/simple-illustrations/simple-illustration__send-money.svg
Comment (shawnborton) [reply assets/images/simple-illustrations/simple-illustration__send-money.svg:1]: Hmm okay, can you try this one: [simple-illustration__sendmoney.svg.zip](https://github.com/Expensify/App/files/15481754/simple-illustration__sendmoney.svg.zip)

And it should get resized down to 48x48. But these all exist in our system at 68x68, so that's what we export them as.

**Comment by jnowakow** [comment]
> @jnowakow Did you replace the icons with the latest shared ones?

`simple-ilustration__sendmoney` is still the one that I exported ([see here](https://github.com/Expensify/App/pull/42184#discussion_r1616823594))

**Comment by shawnborton** [comment]
I think that one is fine as-is, that's how we have it in our system at least.

File: src/languages/es.ts (hunk 2 above)
Comment (luacmartins) [reply src/languages/es.ts:329]: Haha I asked for another translation and got the plural like I suggested above. I think it makes sense to use the plural, since all other items are plural too, e.g. `Expenses`, `Drafts`

File: src/languages/es.ts (hunk 5 above)
Comment (luacmartins) [reply src/languages/es.ts:327]: Bump @jnowakow. I think we should change this to plural since all other ones are plural too

File: src/languages/es.ts (hunk 2 above)
Comment (luacmartins) [reply src/languages/es.ts:329]: Bump @jnowakow. I think we should change this to plural since all other ones are plural too

**Review by luacmartins — CHANGES_REQUESTED** [review]

**Review by luacmartins — APPROVED** [review]
I'm gonna merge this PR to help test the other queries.

**Comment by lanitochka17** [comment]
This PR is failing because of issue https://github.com/Expensify/App/issues/43051

The issue is reproducible in: All environments


https://github.com/Expensify/App/assets/78819774/38c61080-4b36-4d8c-a593-b4906d5adf97

**Comment by Kicu** [comment]
Verified that this is not a problem, as the rest of tabs was explicitly disabled by us here: https://github.com/Expensify/App/pull/43016

**Comment by luacmartins** [comment]
Agree with the comment above
